### DISCUSSION TITLE
Settings > Networks = Error Message

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#132] [Fixed Issues]
@rern

If I click on Settings > Networks, I get an error message.

**Comment by rern** [comment]
Post the message and `networks-data.sh`

**Reply by R101G** [reply]
Errors: JSON.parse: unexpected character at line 1 column 425 of the JSON data▮
{ "bluetooth" : false , "infbt" : false , "inflan" : true , "infwl" : true , "list" : [{ "dhcp" : "static" , "gateway" : "" , "hostname" : "" , "interface": "eth0" , "ip" : "" , "mac" : "b8:27:eb:5b:16:d8" , "ssid" : "" , "dbm" : 0 },{ "dhcp" : "static" , "gateway" : "192.168.1.1" , "hostname" : "rAudio-Pi3B.local" , "interface": "wlan0" , "ip" : "192.168.1.6" , "mac" : "1c:bf:ce:11:18:2c" , "ssid" : "xxxx" , "dbm" : Link Quality=89/100 Signal level=37/100 Noise level=0/100 }] , "hostapd" : false , "hostname" : "rAudio-Pi3B" , "profiles" : [] , "reboot" : ""}

networks-data.sh shows the same information.

**Reply by rern** [reply]
Try:
```sh
wget -O /srv/http/assets/js/networks.js https://github.com/rern/rAudio-1/raw/UPDATE/srv/http/assets/js/networks.js
wget -O /srv/http/bash/networks-data.sh https://github.com/rern/rAudio-1/raw/UPDATE/srv/http/bash/networks-data.sh
```

**Reply by R101G** [reply]
Thanks rern, that seems to have fixed it.

I notice that clicking on the +R logo at top left of the playback page now shows information rather than linking to this site. Is that intentional?
